allensdk/brain_observatory/ecephys/stimulus_table/naming_utilities.py

```diff
@@ -195,4 +195,15 @@ def map_column_names(table, name_map=None, ignore_case=True):
     output = table.rename(columns=name_map)
 
     return output
-#
+
+
+def eval_str(val):
```

Comment:
Does it make sense to add the type suggestions to this function?

Reply:
I don't think so, since it takes in any value of any type and returns any value of any type.

Reply:
But the return types are consistent at least.